client/src/actions/fetchProduct.js

```diff
@@ -0,0 +1,21 @@
+import changeProduct from './productOverview/currentProduct';
+import changeStyles from './productOverview/styleList';
+
+let fetchData = () => {
```

Comment:
I believe we have different API calls, so maybe be more specific about what kind of data we're getting with this call?

Reply:
I agree 👍